### ISSUE TITLE
Translucency

### ISSUE THREAD

**Issue by Answordy** [konwboj/Tumbleweed#22]
I know this might not be an issue (of this mod), sorry for that, however I must ask.
The problem is that the tumbleweeds appaer translucent, even though their texture files have 0% opacity. With vanilla graphics it's not a problem but with shaders they look kinda broken.

The first picture is vanilla, and you can see the kelp right trough it. 
![2022-04-24_05 53 58](https://user-images.githubusercontent.com/82783515/164955982-5c7bc1cc-6d4e-4ca3-8054-9f4d31bd81bc.png)

The second and third pictures are with Complementary shaders, it works the same with AstraLex. Hovewer Sildurs works just fine like vanilla, didn't test other ones. All of these with Optifine however.
![2022-04-24_05 20 28](https://user-images.githubusercontent.com/82783515/164956023-bdb2ff9f-3ca5-41c6-a327-e9f076296211.png)
![2022-04-24_05 20 32](https://user-images.githubusercontent.com/82783515/164956031-85e92dc5-bf5a-4785-9b88-255347abf9f5.jpg)

Most of the layers are also missing as seen on the last picture, making the tumbleweed very.. lean. 
I don't know what causes it, so sorry if it's a shader/optifine problem, I just wondered if there is a way to fix it, or at least remove translucency, if that helps.

**Comment by konwboj** [comment]
Can you try in 1.19?

**Comment by Answordy** [comment]
I did, without shaders the tumbleweed looks the same as before. Somewhat transparent, but not that noticable.
![2022-06-22_11 37 18](https://user-images.githubusercontent.com/82783515/174997647-bec98ccd-a4da-40af-9e37-747a4dd74240.png)
I'm still using 1.18.2 since most mods didn't update to 1.19 yet, one of them is Optifine, what I used when creating the original pictures. So I couldn't recreate the same environment in 1.19. Basically Optifine+Complementary. 
My best bet was to use Rubidim+Oculus+Complementary to test it in the new version. Using that combo the tumbleweeds are totally invisible now.
